include/sam_webgui/roswasm_sam.h

```diff
@@ -50,26 +92,158 @@ class SamActuatorWidget {
 
 class SamDashboardWidget {
 private:
-    bool was_leak;
+    bool was_leak, was_panic = false;
     TopicBuffer<smarc_msgs::Leak>* leak;
+    TopicBuffer<std_msgs::String>* panic;
     TopicBuffer<sensor_msgs::NavSatFix>* gps;
     TopicBuffer<sensor_msgs::BatteryState>* battery;
-    TopicBuffer<nav_msgs::Odometry>* odom;
-    TopicBuffer<sam_msgs::PercentStamped>* vbs;
+    // TopicBuffer<nav_msgs::Odometry>* odom;
+    TopicBuffer<sam_msgs::PercentStamped>* vbs_fb;
     TopicBuffer<sam_msgs::PercentStamped>* lcg;
-    //TopicBuffer<sam_msgs::ThrusterRPMs>* rpms;
-    TopicBuffer<smarc_msgs::ThrusterRPM>* rpm1;
-    TopicBuffer<smarc_msgs::ThrusterRPM>* rpm2;
```

Comment:
Should still be in here?

Reply:
@Jollerprutt

Reply:
No they are now:
TopicWidget<smarc_msgs::ThrusterRPM>* thruster1_rpm;
TopicWidget<smarc_msgs::ThrusterRPM>* thruster2_rpm;